pkg/helmexec/exec.go

```diff
@@ -131,7 +132,19 @@ func New(helmBinary string, enableLiveOutput bool, logger *zap.SugaredLogger, ku
 }
 
 func (helm *execer) SetExtraArgs(args ...string) {
-	helm.extra = args
+	var extraArgs []string
+	
+	// reset the postRenderers and filter the post-renderer args from args and put into helm.postRenderers
+	helm.postRenderers = nil
+	for _, arg := range args {
+		if strings.HasPrefix(arg, "--post-renderer=") || strings.HasPrefix(arg, "--post-renderer-args=") {
```

Comment:
when use a format like the below:
```
--post-renderer  value
```

what will happen?

Reply:
this format will be deal with [analyzeArgs](https://github.com/helmfile/helmfile/blob/main/pkg/argparser/args.go#L57) ， and keep the args  order then past into helm command：
such as :
```
helmfile -e ksp -f ti-k8s/helmfile.yaml -l chart=mysql  --args="--post-renderer=rewrite-ksp.sh"
```

the helm command is : 
```
 helm template mysql mysql --namespace ti-inf --values /tmp/helmfile3923818193/ti-inf-mysql-values-674855cccb --values /tmp/helmfile1141692238/ti-inf-mysql-values-9b4dd874f --values /tmp/helmfile2042720088/ti-inf-mysql-values-5448c6fcbd --values /tmp/helmfile1303814841/ti-inf-acp-mysql-values-776bd65866 --post-renderer rewrite-ksp.sh --debug
```
so these will also work well.

Reply:
@tanguofu 
```
if strings.HasPrefix(arg, "--post-renderer=") || strings.HasPrefix(arg, "--post-renderer-args=") {
```

will `--post-renderer=vaule` `--post-renderer value` all be handled?